src/components/AddBook.jsx

```diff
@@ -45,6 +46,7 @@ const AddBook = () => {
           value={inputState.author}
           onChange={changeState}
           name="author"
+          required="required"
```

Comment:
OPTIONAL
- Instead of giving the attribute a value, it should remains only `required`